File: Algorithms/Warmup/plus_minus.py
# Language: Python 3

def plusMinus(arr):
    positive = 0
    negative = 0
    zero = 0

    for i in range(n):
        if arr[i] > 0:
            positive += 1
        elif arr[i] == 0:
            zero += 1
        else:
            negative += 1
    
    pn = positive/n  
    nn = negative/n
    zn = zero/n

    print(pn)
    print(nn)
    print(zn)

if __name__ == '__main__':
    n = int(input().strip())

    arr = list(map(int, input().rstrip().split()))

    plusMinus(arr)

File: Algorithms/Warmup/plus_minus.js
// Language: JavaScript (Node.js)

let p = 0;
let n = 0;
let z = 0;

for (let i = 0; i < arr.length; i++) {
  if (arr[i] > 0) {
    p += 1;
  } else if (arr[i] == 0) {
    z += 1;
  } else {
    n += 1;
  }
}

console.log(p / arr.length);
console.log(n / arr.length);
console.log(z / arr.length);
